config.py

```diff
@@ -36,15 +35,10 @@ class GG(): pass
 from libraries import amazon_ses
 #from libraries import get_image_size
 
-if os.environ.get('SERVER_SOFTWARE', '').startswith('Dev'):
-	is_sdk=True; is_production=is_appengine=False
-	stripe.verify_ssl_certs = False
-	stripe.api_key=secrets.stripe_test_api_key
-	stripe_pkey=secrets.stripe_test_pkey
-else:
-	is_sdk=False; is_production=is_appengine=True
-	stripe.api_key=secrets.stripe_pkey #secret-key
-	stripe_pkey=secrets.stripe_api_key #publishable-key
+
+is_sdk=False; is_production=is_appengine=True
+stripe.api_key=secrets.stripe_pkey #secret-key
+stripe_pkey=secrets.stripe_api_key #publishable-key
```

Comment:
Not sure we should be removing this is_sdk check, what is the reasoning for this?